test/packages/parallel/gcp/data_stream/compute/_dev/deploy/tf/compute.tf

```diff
@@ -6,17 +6,19 @@ data "google_compute_image" "default" {
 
 resource "google_compute_instance" "default" {
   name = "elastic-package-system-test-${var.TEST_RUN_ID}"
-  // NOTE: e2 instance type is required to collect instance/memory/balloon/* 
+  // NOTE: e2 instance type is required to collect instance/memory/balloon/*
   // metrics, available only on those instances.
   // https://cloud.google.com/monitoring/api/metrics_gcp
   machine_type = "e2-micro"
   zone         = var.zone
 
   labels = {
     run_id = var.TEST_RUN_ID
-    repo_name = var.REPO_NAME
-    pull_request = var.PULL_REQUEST
-    ci_build_number = var.CI_BUILD_NUMBER
+    Environment = "CI"
```

Comment:
Why is this label required? Do we plan to have other environments?

Reply:
I was inclined to search for any cloud resources with `Environment=CI && CreatedDate <= current() -1`, while `Branch`, `Build`, `Owner` are some other labels to help with any troubleshooting or creating some billing dashboards.

Reply:
I added a section in the description (`Implementation details`) to help with the motivation for those labels